src/main/java/UITestFramework/retryLogic/TestListener.java

```diff
@@ -34,36 +34,35 @@ public void onFinish(ITestContext context) {
 		}
 	}
 
-	public void onTestStart(ITestResult result) { 
-
-	}
+	public void onTestStart(ITestResult result) {   }
 
 	public void onTestSuccess(ITestResult result) {   }
    
 	@Override
 	public void onTestFailure(ITestResult result) { 
-	
 		Object TestListener = result.getInstance();
 		WebDriver webDriver = ((CreateSession) TestListener).getDriver();
 		if (webDriver != null)
 		{
-	File scr = ((TakesScreenshot)webDriver).getScreenshotAs(OutputType.FILE);		String filename =  new SimpleDateFormat("yyyyMMddhhmmss'.jpg'").format(new Date());
-    File dest = new File("E:\\" + filename); //Directory where Screenshot get saved.
-   try {
-	FileUtils.copyFile(scr, dest);
-} catch (IOException e) {
-	// TODO Auto-generated catch block
-	e.printStackTrace();
-	}	
+			File scr = ((TakesScreenshot)webDriver).getScreenshotAs(OutputType.FILE);		
+			String filename =  new SimpleDateFormat("yyyyMMddhhmmss'.jpg'").format(new Date());
+    			File dest = new File("E:\\" + filename); //Directory where Screenshot get saved.
```

Comment:
Please provide the `dest` path relative to the framework so it works on all the systems. Please try the below code and see if it creates screenshots at the mentioned destination. Thanks.

`File dest = new File("./Screenshots/" + filename);`